### DISCUSSION TITLE
Deploying Dagster using another CI solution

### DISCUSSION THREAD

**Discussion by doubtingTomNook** [dagster-io/dagster#22693] [Q&A]
I am enjoying Dagster- it makes data pipelines quite easy.
I am having difficulty integrating into existing processes, however- I would like to be able to have my Dagster code to have a similar pipeline to in-house code; i.e.  deployed using CircleCI.
I have tried following the documentation for Dagster CLI, but I am having the following issues:
•	Setting up the configuration:
o	Attempting to follow the instructions for non-interactive setup still leads to the following prompt: `? How would you like to authenticate the CLI? (Use arrow keys)`
o	Attempting to bypass the setup using environment variables and running the serverless command directly doesn’t work either
 
How can I deploy from a CI pipeline other than Github actions? What are the commands I need to run? I have tried looking in the documentation but have come up empty.
 
Thanks!

**Comment by garethbrickman** [comment]
Hey there, just to confirm you're wanting to use CircleCI for [Dagster+ deployment](https://docs.dagster.io/dagster-plus/deployment/serverless#other-bitbucket-or-local-development)? In that case you can create a support ticket from inside the Dagster+ Help menu to get help troubleshooting privately.
